test/fixtures/yarn-3/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "yarn-3",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "lodash": "^4.16.4",
+    "typescript": "^3.9.3"
```

Comment:
Can we drop typescript out of these fixtures? It's a pretty heavyweight dependency and I don't think it's needed for the assertions you wrote. I'm mostly thinking about folks who use this buildpack by cloning from git don't need/want to pay the price of downloading 3 copies of a typescript zip file that is only used in tests.

Reply:
👍 done